src/Doctrine/Repository/PaymentRepositoryTrait.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AdeoWeb\SyliusEveryPayPlugin\Doctrine\Repository;
+
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\NonUniqueResultException;
+use Sylius\Component\Payment\Model\PaymentInterface;
+
+trait PaymentRepositoryTrait
+{
+    /**
+     * @throws NonUniqueResultException
```

Comment:
šitą gal į interface iškelk?

Reply:
Sakai verta?

Reply:
kode vistiek daugiausiai naudojam interface'us, tai šito neiškėlus ten, IDE nerodys, kad šita funkcija gali throwinti tokį exceptioną. už tai susimąsčiau

Reply:
Bet čia ne interfeisas throwina, o implementacija. :)